src/lib/components/PageHeader.svelte

```diff
@@ -0,0 +1,43 @@
+<script lang="ts">
+  export let testId: string | undefined = undefined;
+
+  let hasHeader: boolean;
+  $: hasHeader =
+    $$slots.topLeft !== undefined || $$slots.topRight !== undefined;
```

Comment:
We are generally using `start` and `end` as slot naming convention. 
Inherited from other design system that use such words when they support both RTL and LTR.

Reply:
makes sense, I'll rename

Reply:
done